README.md

```diff
@@ -1 +1,165 @@
+
+## Prerequisites
+
+- LocalStack Pro with the [`localstack` CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli).
+- [Cloud Development Kit](https://docs.localstack.cloud/user-guide/integrations/aws-cdk/) with the [`cdklocal`](https://www.npmjs.com/package/aws-cdk-local) installed.
+- [AWS CLI](https://docs.localstack.cloud/user-guide/integrations/aws-cli/) with the [`awslocal` wrapper](https://docs.localstack.cloud/user-guide/integrations/aws-cli/#localstack-aws-cli-awslocal).
+
+Start LocalStack Pro with the `LOCALSTACK_API_KEY` pre-configured:
+
+```shell
+export LOCALSTACK_API_KEY=<your-api-key>
+DEBUG=1 localstack start
+```
+
+We specified DEBUG=1 to get the printed LocalStack logs directly in the terminal to help us see the event-driven architecture in action. If you prefer running LocalStack in detached mode, you can add the `-d` flag to the `localstack start` command, and use Docker Desktop to view the logs.
+
+## Instructions
+
+You can build and deploy the sample application on LocalStack by running our `Makefile` commands. To deploy the infrastructure, you can run `make deploy` and `make run` after installing the application dependencies. Here are instructions to deploy and test it manually step-by-step.
+
+### Deploying the application
+
+To create the AWS infrastructure locally, you can use CDK and our `cdklocal` wrapper. Before you can deploy the infrastructure, you need to install the application dependencies:
+
+```sh
+yarn
+```
+
+To deploy the infrastructure, you can run the following command:
+
+```sh
+cdklocal bootstrap
+cdklocal deploy --all
+```
+
+This will deploy the `LoanBroker-RecipientList-Stack` and `LoanBroker-PubSub-Stack` stacks. You will see the following output:
+
+```sh
+Outputs:
+LoanBroker-RecipientList-Stack.LoanBrokerArn = arn:aws:states:us-east-1:000000000000:stateMachine:LoanBroker-RecipientList-Stack-LoanBroker641FC9A8-dd79232c
+Stack ARN:
+arn:aws:cloudformation:us-east-1:000000000000:stack/LoanBroker-RecipientList-Stack/e7929928
+```
+
+Take a note of the `LoanBroker-RecipientList-Stack.LoanBrokerArn` output. You will need it to test the application.
+
+## Testing the application
+
+Before you can test the application, you need to pre-populate the `LoanBrokerBanksTable` for the `RecipientsList` stack:
+
+```sh
+awslocal dynamodb put-item \
+    --table-name=LoanBrokerBanksTable \
+    --item='{ "Type": { "S": "Home" }, "BankAddress": {"L": [ { "S": "BankRecipientPremium" }, { "S": "BankRecipientUniversal" }, { "S": "BankRecipientPawnshop" } ] } }'
+```
+
+Start the State Machine execution by running the following command:
```

Comment:
could we add some more explanation here, e.g. outline that this is how we can get quotes from the banks/submit a the loan application?